rest_framework-stubs/viewsets.pyi

```diff
@@ -11,7 +11,7 @@ class ViewSetMixin(object):
     basename: str
     request: Any
     @classmethod
-    def as_view(cls, **initkwargs: Any) -> Callable: ...
+    def as_view(cls, actions: Optional[Dict[str, Any]] = None, **initkwargs: Any) -> Callable: ...
```

Comment:
```suggestion
    def as_view(cls, actions: Optional[Dict[str, str]] = ..., **initkwargs: Any) -> Callable: ...
```

Hey @cs-cordero, can we move forward with this pull request? It'd be really nice to have it merged! You can simply apply the suggested change.

Reply:
My apologies on the delay.  Life got in the way, but I just pushed a new commit to move forward with this request.